engine/src/main/java/org/terasology/rendering/dag/nodes/SimpleBlendMaterialsNode.java

```diff
@@ -74,23 +76,11 @@ public void process() {
      */
     private void preRenderSetupSimpleBlendMaterials() {
         READ_ONLY_GBUFFER.setRenderBufferMask(true, true, true);
-        GL11.glEnable(GL_BLEND);
         GL11.glBlendFunc(GL_SRC_ALPHA, GL_ONE_MINUS_SRC_ALPHA); // (*)
```

Comment:
I'd recommend to remove this method and place its content in process. These preRenderSetup and postRenderCleanup methods made sense when they were full with tricky opengl stuff. Now that they are becoming basically empty they can be removed.